lib/brainstem/presenter.rb

```diff
@@ -150,10 +151,11 @@ def apply_filters_to_scope(scope, user_params, options)
 
       requested_filters = extract_filters(user_params, options)
       requested_filters.each do |filter_name, filter_arg|
-        filter_lambda = configuration[:filters][filter_name][1]
+        filter_lambda = configuration[:filters][filter_name][:value]
 
         args_for_filter_lambda = [filter_arg]
-        args_for_filter_lambda << requested_filters if configuration[:filters][filter_name][0][:include_params]
+        args_for_filter_lambda << requested_filters \
```

Comment:
Rather not to a post-if on the next line.  If you want to wrap, let's not use a post-if.